api-acceptance-tests/src/test/java/com/opencredo/test/api/acceptance/test/config/TestWorld.java

```diff
@@ -0,0 +1,25 @@
+package com.opencredo.test.api.acceptance.test.config;
+
+import com.opencredo.test.api.acceptance.test.interaction.dto.User;
+
+import java.util.HashMap;
+
+public class TestWorld {
+    public String USER_PASSWORD = "Testing123!";
```

Comment:
in this case, I'd probably make this a constant, so `public static final USER_PASSWORD`. You can then refer to it statically so that you don't have to instantiate testworld, e.g. `password = TestWorld.USER_PASSWORD" instead of `testWorld.USER_PASSWORD`. Not an issue for this small test framework but if you have static Utils classes you have to pass instances of testworld in each method to get access to constants